py3dtilers/Common/lod_1.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from ..Common import ObjectToTile
+from scipy.spatial import ConvexHull
+from alphashape import alphashape
+
+
+class FootPrint():
+    def __init__(self, points, min_height, max_height):
+        self.points = points
+        self.min_height = min_height
+        self.max_height = max_height
+
+
+def get_footprint(object_to_tile, override_points=False, other_points=None):
+    geom_triangles = object_to_tile.geom.triangles
+    points = list()
+    minZ = np.Inf
+    average_maxZ = 0
+    for triangles in geom_triangles:
+        maxZ = np.NINF
+        for triangle in triangles:
+            for point in triangle:
+                if len(point) >= 3:
+                    points.append([point[0], point[1]])
+                    if point[2] < minZ:
+                        minZ = point[2]
+                    if point[2] > maxZ:
+                        maxZ = point[2]
+        average_maxZ += maxZ
+    average_maxZ /= len(geom_triangles)
+    if override_points:
+        points = other_points
+    else:
+        # hull = ConvexHull(points)
+        # points = [[points[i][0], points[i][1], minZ] for i in reversed(hull.vertices)]
+        hull = alphashape(points, 0.)
```

Comment:
[alpha shape](https://en.wikipedia.org/wiki/Alpha_shape) creates concave hulls of points. There, I used the [alphashape](https://pypi.org/project/alphashape/) lib, that has a MIT license (maybe @EricBoix can tell us what to do with MIT licenses ?).

Reply:
The [MIT Licence](https://opensource.org/licenses/MIT) is a [popular license](https://opensource.org/licenses) that doesn't protect the owner since it creates no obligation for the user (basically soft is "as is" and with no warranty). MIT License or BSD (among others) are ok for the project.

This info should be placed [in a proper LICENSE file](https://github.com/VCityTeam/py3dtilers/issues/14)